velero-plugins/util/openshift/imageregistry.go

```diff
@@ -35,26 +30,3 @@ func ImageRegistryCapabilityEnabled() (bool, error) {
 	}
 	return false, nil
 }
-
-func ImageRegistryHasReplicas() (bool, error) {
-	c, err := GetImageRegistryConfig()
-	if err != nil {
-		if k8serrors.IsNotFound(err) {
-			return false, nil
-		}
-		return false, err
-	}
-	return c.Status.ReadyReplicas > 0, nil
-}
-
-// https://github.com/openshift/cluster-image-registry-operator/blob/48875d3ccb4595be9d3bec563d1fda2eb940cecf/pkg/defaults/defaults.go#L19
-// avoiding indirect imports and version conflicts
-const ImageRegistryResourceName = "cluster"
-
-func GetImageRegistryConfig() (*v1.Config, error) {
```

Comment:
I think we still need this

If capabilities is enabled we [need another check](https://redhat.enterprise.slack.com/archives/C013VBYBJQH/p1710168733543119?thread_ts=1709952439.633439&cid=C013VBYBJQH) on the registry config.

> it's the managementState (.spec.managementState: Removed) @Tiger, not replicas. Let me see if I can find some docs for reference